frontend/src/components/VideoPreview.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+function VideoPreview({ video }) {
+  return (
+    <div className="video-preview">
```

Comment:
This `div` seems to be a `figure` :eyes: [Documentation](https://developer.mozilla.org/fr/docs/Web/HTML/Element/figure)

Reply:
Resolve Div to Figure  in progress

Reply:
figure should have a figcaption